packages/item/src/vaadin-item-mixin.js

```diff
@@ -38,6 +38,7 @@ export const ItemMixin = (superClass) =>
           value: false,
           reflectToAttribute: true,
           observer: '_selectedChanged',
+          sync: true,
```

Comment:
nit: isn't this change unnecessary since running `performUpdate()` will sync the property change anyway?

Reply:
I guess it's still necessary because the first render is done asynchronously

Reply:
Right, seems unnecessary. Reverted.